src/v2/components/UI/Typography/index.jsx

```diff
@@ -0,0 +1,18 @@
+//@flow
+import React from 'react';
+import cn from 'classnames';
+
+import useStyles from './styles';
+
+type TypographyProps = {
+  type?: 'caption',
+  className?: string,
+};
+
+const Typography = ({type = 'text', className, ...props}: TypographyProps) => {
```

Comment:
This name conflicts with the material-ui/core `Typography` component - can we use an alternate name?

Reply:
@sunnygleason why is it problem at all? linter or build work fine

Reply:
Gave it ```CTypography``` name meaning custom